sdk/monitor/azure-monitor-spring-native/src/main/java/com/azure/monitor/applicationinsights/spring/ConnectionStringRetriever.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.applicationinsights.spring;
+
+import com.azure.monitor.opentelemetry.exporter.implementation.utils.Strings;
+
+import java.util.Optional;
+
+final class ConnectionStringRetriever {
+    private static final String APPLICATIONINSIGHTS_CONNECTION_STRING_ENV =
+            "APPLICATIONINSIGHTS_CONNECTION_STRING";
+
+    private ConnectionStringRetriever() {
+    }
+
+    static Optional<String> retrieveConnectionString(String connectionStringSysProp) {
+
+        if (connectionStringSysProp != null) {
+            if (connectionStringSysProp.isEmpty()) {
+                return Optional.empty();
+            }
+            String connectionStringWithoutDoubleQuotes = connectionStringSysProp.replaceAll("\"", "");
```

Comment:
i'm not sure about special casing here for connection string only, at least let's add comment to explain/justify the special case